rust/processor/src/models/token_v2_models/v2_token_utils.rs

```diff
@@ -412,6 +341,7 @@ impl PropertyMapModel {
     }
 }
 
+// TODO: rename this to ObjectResource
```

Comment:
Some of these are actually token specific. The right thing to do is to have ObjectCore be split out to a V2ObjectResource and everything else stays here.